crypto/pemutil/pem.go

```diff
@@ -165,6 +165,47 @@ func ReadCertificate(filename string) (*realx509.Certificate, error) {
 	return crt, errors.Wrapf(err, "error parsing %s", filename)
 }
 
+// ReadCertificateBundle returns a list of *x509.Certificate from the given
+// filename. It supports certificates formats PEM and DER. If a DER-formatted
+// file is given only one certificate will be returned.
+func ReadCertificateBundle(filename string) ([]*realx509.Certificate, error) {
+	b, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, errs.FileError(err, filename)
+	}
+
+	// PEM format
+	if bytes.HasPrefix(b, []byte("-----BEGIN ")) {
+		var block *pem.Block
+		var bundle []*realx509.Certificate
```

Comment:
since you're here we could probably just change all the `realx509` -> `x509`, and `x509` -> `stepx509`. shrug

Reply:
Fixed with 4d2bd33c8d5ffb8735e2629505e5b7bce2375b2b